api/src/rateLimiters.js

```diff
@@ -0,0 +1,39 @@
+class RateLimiter {
+  constructor(limitPerSecond) {
```

Comment:
To avoid having errors if it's `undefined` you might want to set a default value, for example `10`

```suggestion
  constructor(limitPerSecond = 10) {
```